stratum/hal/lib/tdi/es2k/es2k_sde_table_data.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+// Target-agnostic SDE wrapper Table Data methods.
+
+#include <string>
+
+#include "stratum/hal/lib/tdi/tdi_constants.h"
+#include "stratum/hal/lib/tdi/tdi_pkt_mod_meter_config.h"
+#include "stratum/hal/lib/tdi/tdi_sde_helpers.h"
+#include "stratum/hal/lib/tdi/tdi_sde_wrapper.h"
+
+namespace stratum {
+namespace hal {
+namespace tdi {
+
+using namespace stratum::hal::tdi::helpers;
+
+::util::Status TableData::SetPktModMeterConfig(
+    const TdiPktModMeterConfig& cfg) {
+  if (cfg.isPktModMeter) {
+    RETURN_IF_ERROR(SetField(table_data_.get(), kMeterCirPps, cfg.cir));
+    RETURN_IF_ERROR(
+        SetField(table_data_.get(), kMeterCommitedBurstPackets, cfg.cburst));
+    RETURN_IF_ERROR(SetField(table_data_.get(), kMeterPirPps, cfg.pir));
+    RETURN_IF_ERROR(
+        SetField(table_data_.get(), kMeterPeakBurstPackets, cfg.pburst));
+  } else {
```

Comment:
Add a comment saying this section is for direct

Reply:
Added comment

Reply:
I see it in only one place. You might have missed this one